### ISSUE TITLE
Incorrect reference to desktop:Extension should be Uap:Extension

### ISSUE THREAD

**Issue by TimMangan** [MicrosoftDocs/winrt-related#322]
[Enter feedback here]
In the Remarks section on this page (https://learn.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-desktop7-approvedshellextension) there is an IMPORTANT box with text describing some requirements.  

The text includes:
 **You set the desktop7:Scope attribute on the [desktop:Extension](https://learn.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-desktop-extension) element.**

I believe that the parent application extension element that holds the desktop7:Scope attribute must be a Uap:Extension element. Or maybe something else.

The scope attribute is defined in Desktop7 schema, but is not attached to the Desktop7 Application Extension element. It is only referenced in the PreviewManifestSchema_MsixAppCompatSupport through PreviewManifestSchema_MsixAppCompatSupport_V3 schemas.  These schemas define a previewappcompat:PreviewEntryPointAttributesGroup that is only referenced in the UAP schema.  Note that the UAP10 schema references the original preview and not V3, although the usage in an appxmanifest file appears the same, we also have to be careful about the namespaces listed in the appxmanifest..

As this gets resolved, if there is an OS build version dependency for using this, it should be included.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: c6b5e0ec-37f6-ee99-bff5-bf7ef321ec93
* Version Independent ID: c6b5e0ec-37f6-ee99-bff5-bf7ef321ec93
* Content: [desktop7:ApprovedShellExtension - Windows UWP applications](https://learn.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-desktop7-approvedshellextension)
* Content Source: [winrt-related-src/schemas/appxpackage/uapmanifestschema/element-desktop7-approvedshellextension.md](https://github.com/MicrosoftDocs/winrt-related/blob/docs/winrt-related-src/schemas/appxpackage/uapmanifestschema/element-desktop7-approvedshellextension.md)
* Product: **uwp**
* Technology: **winrt-reference**
* GitHub Login: @drewbatgit
* Microsoft Alias: **drewbat**

**Comment by TimMangan** [comment]
Possibly it should be Desktop7.  Needless to say, it is confusing!

**Comment by drewbatgit** [comment]
Hello, thanks for following up on this. We recently made a big update related to desktop7:scope that hopefully made the current documentation correct, if still confusing. desktop7:scope is declared in the desktop7 schema but then is only directly referenced in the ManganeseExtensionAttributesGroup, which is in turn referenced by a lot (~40) of extensions. We recently went through and added the ManganeseExtensionAttributesGroup attributes to all of the extension pages that reference it, including desktop:Extension. So, I believe that the important note about the adding desktop7:scope to your desktop:Extension is correct. I'm going to close the issue, but feel free to go ahead and reply if you feel I haven't addressed the issue.
